src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs

```diff
@@ -97,12 +98,15 @@ public static RazorConfiguration SelectConfiguration(Version version)
             LanguageVersion = languageVersion;
             ConfigurationName = configurationName;
             Extensions = extensions;
+            UseConsolidatedMvcViews = useConsolidatedMvcViews;
```

Comment:
Hmm, how should we be consuming this at the tooling layer?

Reply:
I believe tooling is largely unaffected here.

We added this API to support the unified assembly scenario where the Razor source generator compiles both views and application types into a single assembly. As part of this, we need to add a way to make the view types generated by the Razor compiler internal and sealed by default (instead of public).

AFAIK, since the tooling consumes a completely separate design time representation, it doesn't need to React to this.

Let me know if I missed some nuance to this.

Reply:
Probably should be fine. Might want to validate that the existing editor doesn't fall over in projection scenarios but that's about it